web/src/beta/features/Editor/index.tsx

```diff
@@ -32,7 +33,8 @@ const Editor: React.FC<Props> = ({ sceneId, tab }) => {
               {leftPanel}
             </Resizable>
           )}
-          <Center>
+          <Center hasStory={isStory}>
+            {isStory && <StoryPanel />}
```

Comment:
I feel like the StoryPanel is in the same category as widgets and the infobox. So at the very least I think it should go in to the Visualizer, but it might make sense to put it in core (or move the widgets and infobox out of core??)
Let's discuss with @keiya01 about this next week